front/src/applications/editor/tools/selection/components.tsx

```diff
@@ -32,11 +32,11 @@ export const SelectionMessages = () => {
 };
 
 const SelectionZone = ({ newZone }: { newZone?: Zone }) =>
-  newZone ? (
+  newZone && (
```

Comment:
return early

Reply:
I moved the condition in the parent component, done